modules/headline.py

```diff
@@ -12,16 +14,18 @@ def __init__(self):
 
     def extract_headline(self):
         self.net_con=True #Expecting Internet Connection to be working initially
```

Comment:
not necessary

Reply:
It was a part of the original code, but removed.